common/inc/stdc++/linux/exception

```diff
@@ -33,11 +33,11 @@ namespace std
 	class exception
 	{
 	public:
-		exception() throw();
```

Comment:
I don't get failures either, so not necessary. But they do have exception specifications, and they are deprecated.